ts/features/pn/aar/utils/detailsById.ts

```diff
@@ -0,0 +1,7 @@
+import { MessageBodyMarkdown } from "../../../../../definitions/backend/MessageBodyMarkdown";
+import { UIMessageDetails } from "../../../messages/types";
+
+export const fillerEphemeralAARMarkdown =
```

Comment:
See comment above of not using the fake markdown as an identifier. Consequently, I would avoid declaring an util and move this code into the only production file that uses it, repeating it, if needed, in the test